src/main/index.ts

```diff
@@ -151,9 +151,17 @@ app.on("ready", async () => {
   extensionLoader.init();
   extensionDiscovery.init();
 
+  // Start the app without showing the main window when auto starting on login
+  // (On Windows and Linux, we get a flag. On MacOS, we get special API.)
+  const startHidden = process.argv.includes("--hidden") || (isMac && app.getLoginItemSettings().wasOpenedAsHidden);
```

Comment:
Is it electron's flag or specified by Lens? (if by lens => should we collect all available cli flags in some common place, e.g. `vars.ts`?)
Is it different from `--process-start-args: --hidden`? (found in docs [here](https://www.electronjs.org/docs/api/app#appsetloginitemsettingssettings-macos-windows))